ZipBuilder/Sources/ZipBuilder/ModuleMapBuilder.swift

```diff
@@ -21,14 +21,16 @@ import Foundation
 struct ModuleMapBuilder {
   /// Information associated with a framework
   private class FrameworkInfo {
-    let isOpenSource: Bool
-    let podInfo: CocoaPodUtils.PodInfo
+    let isSourcePod: Bool
+    let installedLocation: URL
+    let versionedPod: CocoaPodUtils.VersionedPod
```

Comment:
Isn't this represented in a `PodInfo` type?

Reply:
Got it... I was thinking that the `PodInfo` contains both `installedLocation` and the `VersionedPod` so I wasn't sure why we were using the types separately. Thanks!